shared_steps/mailslurper.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+var request = require('request-promise');
+
+const options = {
+    method: 'GET',
+    uri: 'http://localhost:8085/mail',
+    json: true,
+
+}
+
+const getEmailSubject = (email) => request(options)
+    .then(function(response) {
+        var data = response.mailItems;
+        var object = data.filter(d => d.toAddresses.includes(email))[0];
```

Comment:
unclear variables. Change to 
`const responseData =`
and
`const filteredAddresse = `or something similar